docs/learn/overview/intro.mdx

```diff
@@ -2,4 +2,28 @@
 slug: /
 ---
 
-# Welcome to the Lit Protocol Docs
\ No newline at end of file
+# Welcome to the Lit Protocol Docs
+
+You are viewing the _Learn_ section of the Lit Protocol documentation. This section is aims to teach the various Lit Protocol concepts and how to use them.
+
+If you are looking for information on how to build on Lit Protocol, please visit the [Build section](/build) of the documentation.
+
+## Getting Started with Lit
+
+If you are completely new to Lit, we recommend starting with the [What is Lit Protocol](./what-is-lit) page, followed by the [How Lit Protocol Works](./how-it-works) page. These pages will provide you with a high-level overview of Lit Protocol and what you can do with it.
```

Comment:
I'm assuming these links will be updated later, just leaving a note here to make sure they aren't forgotten

Reply:
* most links in the page